compiler/rustc_parse/src/parser/mod.rs

```diff
@@ -211,9 +211,10 @@ pub type ReplaceRange = (Range<u32>, Vec<(FlatToken, Spacing)>);
 /// Controls how we capture tokens. Capturing can be expensive,
 /// so we try to avoid performing capturing in cases where
 /// we will never need an `AttrTokenStream`.
-#[derive(Copy, Clone)]
+#[derive(Copy, Clone, Default)]
```

Comment:
Is this default used anywhere? If not, please let's keep this explicit. I think "this is the default mode" doesn't mean "this needs a `Default` impl", it just means "this is what's used in `Parser::new`").

Reply:
Mmkay!